crates/transaction-pool/src/validate/constants.rs

```diff
@@ -8,7 +8,7 @@ pub const TX_SLOT_BYTE_SIZE: usize = 32 * 1024;
 /// non-trivial consequences: larger transactions are significantly harder and
 /// more expensive to propagate; larger transactions also take more resources
 /// to validate whether they fit into the pool or not.
-pub const MAX_TX_INPUT_BYTES: usize = 4 * TX_SLOT_BYTE_SIZE; // 128KB
+pub const DEFAULT_MAX_TX_INPUT_BYTES: usize = 4 * TX_SLOT_BYTE_SIZE; // 128KB
```

Comment:
@emhane thoughts on naming? :)

Reply:
reason i choose the naming is to avoid this const accidently used by other modules where max_tx_input_bytes should be used

Reply:
default indicates this is not a fixed max. a fixed max would be a value that is spec'd out or has been found by benchmarking. ultimately, all default constants are eventually exposed as cli configurable parameters.

Reply:
oh, i just happen to add the default since it feels the most sensible choice after exposing to cli but didn't know the rule before. so is it a spec'd limit on how large a tx can be?